en/includes/references/grant-types.md

```diff
@@ -1,49 +1,111 @@
-
-This helps to protect the user's credentials and prevents them from being compromised by malicious client applications.
+The Authorization code flow is a secure method for clients to obtain an access token by exchanging an authorization code received through the front channel. As the access token is directly transmitted to the client via a secure back channel, it is not exposed to the user's user agent (e.g. web browser) reducing its risk of being intercepted by a malicious party.
 
 The following diagram shows how the authorization code flow works.
 
 ![How the authorization code grant works]({{base_path}}/assets/img/references/grants/authorization-code.png)
 
 1. The user visits the client application and requests for login through {{ product_name }}.
-2. The client application redirects the authorization code request to {{ product_name }}.
+2. The client application redirects the authorization code request to {{ product_name }} (front channel).
+
+
+    === "Request format (/authorize)"
+
+        ``` bash
+        {{base_url}}/oauth2/authorize
+        ?response_type=code
+        &client_id=<CLIENT_ID>
+        &redirect_uri=<REDIRECT_URI>
+        ```
+
+    === "Sample request (/authorize)"
+
+        ```bash
+        {{base_url_example}}/oauth2/authorize
+        ?response_type=code
+        &client_id=cHuGSLCm77ApRFcsdyh_4sdFU2XYa
+        &redirect_uri=https://localhost:3000
+        ```
+
 3. {{ product_name }} prompts the user to enter credentials.
 4. The user enters the credentials.
 5. After successful authentication, {{ product_name }} sends the authorization code to the client application.
-6. The client application uses this authorization code to request an access token from {{ product_name }}.
-7. {{ product_name }} sends the access token and ID token to the client application.
+
+    ``` bash
+    http://localhost:3000?code=9142d4cad58c66d0a5edfad8952192
+    ```
+6. The client application uses this authorization code to request an access token from {{ product_name }} (back channel).
+
+    === "Request format (/token)"
+
+        ``` bash
+        curl -v -X POST {{base_url}}/oauth2/token\ 
+        --basic -u <CLIENT_ID>:<CLIENT_SECRET> \
+        --header "Content-Type:application/x-www-form-urlencoded;charset=UTF-8" -k \
+        --data-urlencode "grant_type=authorization_code" \
+        --data-urlencode "code=<AUTHORIZATION_CODE>" \
+        --data-urlencode "redirect_uri=<REDIRECT_URI>"
+        ```
+
+    === "Sample request (/token)"
+
+        ```bash
+        curl -v -X POST {{base_url_example}}/oauth2/token\ 
+        --basic -u Ei0Wf9bzfsdvdd0lfwSvTPje8kcEa:HFI2S2HR9kdfg543f5xMpuIAjgJxnZQUT1_e52ga \
+        --header "Content-Type:application/x-www-form-urlencoded;charset=UTF-8" -k \
+        --data-urlencode "grant_type=authorization_code" \
+        --data-urlencode "code=0sdvsw1-05a2-3ebe-bb42-e007160b46f4" \
```

Comment:
```suggestion
        --data-urlencode "code=9142d4cad58c66d0a5edfad8952192" \
```
Let's use the same code mentioned as received authz code. Otherwise, reader is confusing on the sample values